backend/src/utils/stream.py

```
@@ -286,3 +290,61 @@ async def stream_generator(
+
+                    if b"done" in data:
+                        yield "data: [DONE]\n\n"
+                        return
+                    if b"error" in data:
+                        error_msg = data[b"error"].decode()
+                        yield f'data: {{"error": "{error_msg}"}}\n\n'
+                        yield "data: [DONE]\n\n"
+                        return
+                    if b"data" in data:
+                        yield f"data: {data[b'data'].decode()}\n\n"
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)